crates/sim2h/src/lib.rs

```diff
@@ -626,46 +628,51 @@ async fn handle_message_join_space(
 ) {
     sim2h_handle
         .state()
+        .clone()
```

Comment:
uhhh - if we clone the state here the new connection will get added to the clone and not the actual state - then get dropped.

Reply:
scratch that - I see this is cloning the handle not the state : )